app/src/main/java/com/babylon/wallet/android/presentation/accessfactorsources/derivepublickey/DerivePublicKeyViewModel.kt

```diff
@@ -173,7 +190,7 @@ class DerivePublicKeyViewModel @Inject constructor(
     }
 
     sealed interface Event : OneOffEvent {
-        data object RequestBiometricPrompt : Event
+        data class RequestBiometricPrompt(val onlyDeviceCredentials: Boolean = false) : Event
```

Comment:
I am a bit confused here. Do you really use the `onlyDeviceCredentials` somewhere?
It is always false in this viewmodel

Reply:
yes, that I added in very early iteration as possible solution to the issue, to ask user to authenticate only with credentials. But it feel risky to base this logic on our assumptions about secure folder, without actual fact based logic why we experience those problems, this is why I don't use it. Will remove